src/plugins/intel_cpu/tests/unit/transformations/move_eltwise_up_data_movement_test.cpp

```diff
@@ -271,3 +271,73 @@ TEST_F(MoveEltwiseUpThroughDataMovTest, SingleUnaryEltwiseDynamicRank) {
+                                                          255);
+        auto transpose_const = ov::opset8::Constant::create(ov::element::i64, ov::Shape{input_order.size()}, input_order);
+        auto transpose = std::make_shared<ov::opset8::Transpose>(fakequantize, transpose_const);
+
+        model_ref = std::make_shared<ov::Model>(ov::NodeVector{transpose}, ov::ParameterVector{input});
+    }
+}
+
+TEST_F(MoveEltwiseUpThroughDataMovTest, TransposeFakeQuantizePerChannel) {
+    const ov::Shape shape{1, 3, 12, 64};
+    const std::vector<int64_t> input_order = {0, 2, 1, 3};
+    {
+        auto input = std::make_shared<ov::opset8::Parameter>(ov::element::f32, shape);
+
+        auto transpose_const = ov::opset8::Constant::create(ov::element::i64, ov::Shape{input_order.size()}, input_order);
+        auto transpose = std::make_shared<ov::opset8::Transpose>(input, transpose_const);
+
+        auto fakequantize = std::make_shared<ov::opset8::FakeQuantize>(input,
+                                                          ov::opset8::Constant::create(ov::element::f32, ov::Shape{1, 3, 1, 1}, {-8.5, -7.5, -10.}),
+                                                          ov::opset8::Constant::create(ov::element::f32, ov::Shape{1, 3, 1, 1}, {8.5, 7.5, 10.}),
+                                                          ov::opset8::Constant::create(ov::element::f32, ov::Shape{}, {-128}),
+                                                          ov::opset8::Constant::create(ov::element::f32, ov::Shape{}, {127}),
+                                                          255);
+
+        model = std::make_shared<ov::Model>(ov::NodeVector{fakequantize}, ov::ParameterVector{input});
+        manager.register_pass<ov::intel_cpu::MoveEltwiseUpThroughDataMov>();
+    }
+    {
+        auto input = std::make_shared<ov::opset8::Parameter>(ov::element::f32, shape);
+
+        auto transpose_const = ov::opset8::Constant::create(ov::element::i64, ov::Shape{input_order.size()}, input_order);
+        auto transpose = std::make_shared<ov::opset8::Transpose>(input, transpose_const);
+
+        auto fakequantize = std::make_shared<ov::opset8::FakeQuantize>(input,
+                                                          ov::opset8::Constant::create(ov::element::f32, ov::Shape{1, 3, 1, 1}, {-8.5, -7.5, -10.}),
+                                                          ov::opset8::Constant::create(ov::element::f32, ov::Shape{1, 3, 1, 1}, {8.5, 7.5, 10.}),
+                                                          ov::opset8::Constant::create(ov::element::f32, ov::Shape{}, {-128}),
+                                                          ov::opset8::Constant::create(ov::element::f32, ov::Shape{}, {127}),
+                                                          255);
+
+        model_ref = std::make_shared<ov::Model>(ov::NodeVector{fakequantize}, ov::ParameterVector{input});
+    }
```

Comment:
In case of negative test cases (when model is not changed by the transformation) we don't need to build `model_ref` manually - it is done automatically by the tests class

Reply:
Removed reference model.